env_handler.go

```diff
@@ -2,11 +2,12 @@ package main
 
 import (
 	"github.com/blaines/tasque-go/result"
```

Comment:
this seems to be a hangover from before we forked... why are we importing from a different repo when there we have our own `result` stuct here https://github.com/Skycatch/tasque-go/blob/master/result/result.go?

Reply:
agreed. Even it can be a relative path.